introduction.md

```diff
@@ -156,6 +156,17 @@ These people work at:
 
 It is not aimed at public organizations' end users (residents or citizens), journalists or academics.
 
+### What does this not do
+
+The Standard for Public Code is not meant to cover individual implementations of a codebase.
+As such, it does not create demands on the local technical infrastructure or legal framework of a given public organization.
+
+Also, while the Standard for Public Code refers to several standards and have considerable overlap with others, it has the focus to enable collaboration.
+Therefore, it does not aim to replace quality standards, like the ISO 25000 series, or those focusing on security, like the [OpenSSF Best Practices Badge](https://github.com/coreinfrastructure/best-practices-badge), but to synergize well with them.
+
+And while the focus is to enable collaboration, it will not guarantee that a community will spring into existence.
```

Comment:
If we change from focus to purpose, I'd propose that it should be "purpose includes enabling" because the purpose of the standard is broader than just enabling collaboration.